tests/h264Testing/h264Verifier.js

```diff
@@ -0,0 +1,95 @@
+const Promise = require('bluebird');
+const exec = require('child_process').exec;
+const fs = require('fs');
+require('../../lib/utils/KalturaConfig');
+require('../../lib/utils/KalturaCache');
+
+class h264Verifier
+{
+	/**
+	 * @param flavorIds - array of flavor ids
+	 * @param flavorPaths - array of flavor paths
+	 */
+	static insertFlavorsPathToCache(flavorIds, flavorPaths)
+	{
+		h264Verifier.writeToLog(`called insertFlavorsPathToCache with ${flavorIds} and ${flavorPaths}`, 'h264.log');
+		const testTime = KalturaConfig.config.h264Verfication.cacheTime;
+		for (let i = 0; i < flavorIds.length; i++)
+		{
+			KalturaCache.set(flavorIds[i], { path: flavorPaths[i] }, testTime,

```

Comment:
Did you verify this in memcached as well , I think there is a problem with inserting JSON as is - please verify